### ISSUE TITLE
Error for certain streamers

### ISSUE THREAD

**Issue by Astrobinary** [ttvintrusiveadblocker/ttv-intrusive-blocker#5]
The plugin seems to be causing issues with random streamers. It is not consitant on what streamer gives the error, but for the most part below are the ones I have been having errors with.

XQCOW - This video is either unavailable or not supported in this browser. (Error #4000)
sodapoppin - There was a network error. Please try again. (Error #2000)
mizkif - There was a network error. Please try again. (Error #2000)

**Comment by ItsBluey** [comment]
same here disabling then refreshing fixes then enable again

**Comment by ImDini** [comment]
> same here disabling then refreshing fixes then enable again i learnt but this probably easy fix

This worked for me first time but now it isn't

**Comment by ttvintrusiveadblocker** [comment]
Think it has more to do with all the other issues twitch has been facing the last few days https://status.twitch.tv/ all last night they were having problems with all sorts of things on the site. People without any extensions were also having the same issues.

**Comment by PurpleHerby** [comment]
https://www.twitch.tv/quin69 with add-on enabled I get error 2000, disable add-on & refresh, works fine. 
Can repeat as many times as I want at the moment.

**Comment by ttvintrusiveadblocker** [comment]
Yeah you are right, seems to be an issue with the cache, looks like if someone goes to the stream before it starts, it will cache an m3u8 playlist that doesn't work for an hour. I have disabled the cache until we can work out a way to verify it.

**Comment by PurpleHerby** [comment]
Glad you found the cause. nice job on the addon in general btw.

**Comment by ImDini** [comment]
> Yeah you are right, seems to be an issue with the cache, looks like if someone goes to the stream before it starts, it will cache an m3u8 playlist that doesn't work for an hour. I have disabled the cache until we can work out a way to verify it.

Appreciate the work on this! Saving me from buying another month of turbo till things 'work out' lol

**Comment by ImDini** [comment]
Seems to be happening again

**Comment by johnpyp** [comment]
Just broke for me as well. If you follow the HLS link to the cloudflare worker, says "Error 1101: Worker threw exception"

**Comment by N3ars** [comment]
I don't think it's twitch's fault this time but cloudflare server...
